src/main/java/duke/InvalidCommandException.java

```diff
@@ -0,0 +1,4 @@
+package duke;
+
+public class InvalidCommandException extends Exception{
```

Comment:
It is good that new exception classes are created for different purpose. However, in this case, it currently seems to be doing the same thing as its parent class, `Exception`. Consider revising empty classes by removing them or updating them accordingly.